rskj-core/src/test/java/co/rsk/peg/lockingcap/LockingCapSupportImplTest.java

```diff
@@ -0,0 +1,303 @@
+        // Arrange
+        activations = ActivationConfigsForTest.wasabi100().forBlock(0);
+        lockingCapSupport = new LockingCapSupportImpl(lockingCapStorageProvider, activations, constants, signatureCache);
+
+        // Act
+        Optional<Coin> actualLockingCap = lockingCapSupport.getLockingCap();
+
+        // Assert
+        assertEquals(Optional.empty(), actualLockingCap);
+    }
+
+    @Test
+    void increaseLockingCap_whenNotExistsPreviousValueAndTakeDefaultInitialValueAndNewLockingCapIsGreaterThanInitialValue_ShouldReturnTrue() {
+        // Arrange
+        Coin newLockingCap = constants.getInitialValue().add(Coin.SATOSHI);
+        Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, LockingCapCaller.AUTHORIZED.getRskAddress());
+
+        // Act
+        boolean actualResult = lockingCapSupport.increaseLockingCap(tx, newLockingCap);
+
+        // Assert
+        assertTrue(actualResult);
+        assertEquals(Optional.of(newLockingCap), lockingCapSupport.getLockingCap());
+    }
+
+    @Test
+    void increaseLockingCap_whenNotExistsPreviousValueAndTakeDefaultInitialValueAndNewLockingCapIsLessThanInitialValue_ShouldReturnFalse() {
+        // Arrange
+        Coin newLockingCap = constants.getInitialValue().subtract(Coin.SATOSHI);
+        Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, LockingCapCaller.AUTHORIZED.getRskAddress());
+
+        // Act
+        boolean actualResult = lockingCapSupport.increaseLockingCap(tx, newLockingCap);
+
+        // Assert
+        assertFalse(actualResult);
+        assertEquals(Optional.of(constants.getInitialValue()), lockingCapSupport.getLockingCap());
+    }
+
+    @Test
+    void increaseLockingCap_whenExistInStorageAPreviousValueAndNewLockingCapIsGreaterThanPreviousValue_ShouldReturnTrue() {
+        // Arrange
+        Coin previousLockingCap = constants.getInitialValue().add(Coin.SATOSHI);
+        lockingCapStorageProvider.setLockingCap(previousLockingCap);
+        lockingCapSupport.save();
+        // Recreate LockingCapSupport to load the previous value from storage
+        lockingCapStorageProvider = new LockingCapStorageProviderImpl(bridgeStorageAccessor);
+        lockingCapSupport = new LockingCapSupportImpl(lockingCapStorageProvider, activations, constants, signatureCache);
+
+        Coin newLockingCap = previousLockingCap.add(Coin.SATOSHI);
+        Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, LockingCapCaller.AUTHORIZED.getRskAddress());
+
+        // Act
+        boolean actualResult = lockingCapSupport.increaseLockingCap(tx, newLockingCap);
+
+        // Assert
+        assertTrue(actualResult);
+        assertEquals(Optional.of(newLockingCap), lockingCapSupport.getLockingCap());
+    }
+
+    @Test
+    void increaseLockingCap_whenExistInStorageAPreviousValueAndNewLockingCapIsLessThanPreviousValue_ShouldReturnFalse() {
```

Comment:
Similar renaming for all other test names. What do you think?

Reply:
Done! Applied the same method name pattern for the rest of the tests.